src/Microsoft.Azure.WebJobs/HandlesReturnValueAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Description
+{
+    /// <summary>
+    /// 
```

Comment:
>/// [](start = 4, length = 3)

I would suggest: "Indicates that a trigger binding is capable of handling return values from a function." or something along those lines.